disassemblers/ofrak_binary_ninja/ofrak_binary_ninja/components/binary_ninja_analyzer.py

```diff
@@ -15,6 +21,21 @@
 LOGGER = logging.getLogger(__file__)
 
 
+class _BinjaExternalTool(ComponentExternalTool):
```

Comment:
One other concern I have about making Binary Ninja an external tool like this is that it's not consistent with the other analysis back ends. We only show information about when Binary Ninja is missing, but don't show any of the respective information about which components won't be loaded when we don't have Ghidra installed, , for example.

Reply:
Note that this is here because the binja backend is the one that actually prevents OFRAK from working at all if binja is not installed. Other backends do not have the issue.